package.json

```
@@ -28,7 +28,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/awesomejerry/react-native-qrcode-svg.git"
+    "url": "git+https://github.com/Expensify/react-native-qrcode-svg.git"
```

```suggestion
    "url": "git+ssh://github.com/Expensify/react-native-qrcode-svg.git"
```

Btw shouldn't it be like this?

---

We have it without ssh in our other repos, e.g. https://github.com/Expensify/react-native-live-markdown/pull/622/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L50

---

makes sense, thanks